tests/unittests/netmap/metadata_json_test.py

```diff
@@ -21,9 +21,9 @@
 from nav.models.profiles import NetmapViewNodePosition
 from nav.netmap import stubs, metadata
 from nav.netmap.metadata import Edge, Group
-from metaclass_testcase import MetaClassTestCase
-from topology_layer3_testcase import TopologyLayer3TestCase
-from topology_layer2_testcase import TopologyLayer2TestCase
+from .metaclass_testcase import MetaClassTestCase
+from .topology_layer3_testcase import TopologyLayer3TestCase
+from .topology_layer2_testcase import TopologyLayer2TestCase
 
```

Comment:
I consider this syntax change a big win for readability. Thank you, py3k

Reply:
Not to mention all the potential conflicts that could happen without it